foundations/src/telemetry/log/init.rs

```diff
@@ -70,6 +71,9 @@ pub(crate) fn init(service_info: &ServiceInfo, settings: &LoggingSettings) -> Bo
     // NOTE: OXY-178, default is 128 (https://docs.rs/slog-async/2.7.0/src/slog_async/lib.rs.html#251)
     const CHANNEL_SIZE: usize = 1024;
 
+    // buffer json log lines up to 500 characters
+    const BUF_SIZE: usize = 500;
```

Comment:
With lots of keys in json it's easy to exceed this. I'd go with something like 4k.

Reply:
Sorry, didn't see this last night and it didn't go to my cloudflare email for some reason even though I added it to this account. Thanks again!